docs/HowToDebugCrashWithAddr2lineTool.md

```diff
@@ -0,0 +1,116 @@
+# How to use Addr2Line tool for debugging
+**addr2line** is a tool which comes with the Toolchain in Linux.
+It converts the given address into file names and line numbers.  
+Developers can debug the TizenRT ELF/AXF output file through **addr2line**.
+It is escpecially helpful which debugging a crash and can be used to find the 
```

Comment:
Please change `escpecially` to `especially`.

Reply:
ok